### DISCUSSION TITLE
GoodbyeDPI UI v1.2.2

### DISCUSSION THREAD

**Discussion by Storik4pro** [Storik4pro/goodbyeDPI-UI#30] [Announcements]
### [RU]
>[!Important]
>Вы можете обсудить нововведения этого обновления в [соответствующей дискуссии](https://github.com/Storik4pro/goodbyeDPI-UI/discussions/24)

## Что нового в этом обновлении?

- ✨ Теперь вы можете посмотреть списки сайтов, которые используются в выбранном пресете (предложение #13)
- ✨ Теперь можно изменить поставщика для `russia-blacklist.txt` (предложение #14)

## БЕТА - версии функций 

- 🛠 Теперь вы можете изменять настройки стандартных пресетов с помощью кнопки `"Изменить"` рядом с меню выбора пресета. **Обратите внимание:** Данная функция доступна только для zapret. При обновлении компонента/UI все измененные данные будут потеряны

## Список исправлений

- Теперь статус процесса на домашней странице отображается корректно при автозапуске (#28)
- Теперь сообщение `"Доступны новые обновления"` в меню компонентов не ошибочно будет отображаться для goodbyedpi (#26)
- Проверка наличия обновлений для компонентов была оптимизирована
- Были возвращены уведомления при запуске процесса. Вы все ещё можете отключить их в меню настройки уведомлений

### [EN]

>[!Important]  
>You can discuss the new features in this update in [the relevant discussion](https://github.com/Storik4pro/goodbyeDPI-UI/discussions/24)

## What's new in this update?

- ✨ You can now view the list of sites used in the selected preset (suggestion #13).
- ✨ You can now change the provider for `russia-blacklist.txt` (suggestion #14).

## BETA features

- 🛠 You can now modify the settings of standard presets using the "Edit" button next to the preset selection menu. **Note:** This feature is available only for zapret. Any changes will be lost when the component/UI is updated.

## List of fixes

- The process status on the home page now displays correctly on autostart (#28).
- The "New updates available" message in the components menu will no longer incorrectly display for goodbyedpi (#26).
- Update checking for components has been optimized.
- Notifications upon process startup have been reinstated. You can still disable them in the notifications settings menu.

<hr /><em>This discussion was created from the release <a href='https://github.com/Storik4pro/goodbyeDPI-UI/releases/tag/1.2.2'>GoodbyeDPI UI v1.2.2</a>.</em>

**Comment by LordArrin** [comment]
Жду релиз в microsoft store 🥇

**Reply by LordArrin** [reply]
А так же настройку "сворачивать в трей при закрытии".

**Reply by Storik4pro** [reply]
> Жду релиз в microsoft store 🥇

Маловероятно. Приложение не пройдет некоторые проверки

> А так же настройку "сворачивать в трей при закрытии".

Хорошо, добавлю

**Reply by Storik4pro** [reply]
> А так же настройку "сворачивать в трей при закрытии".

Было добавлено в [обновлении 1.2.3](https://github.com/Storik4pro/goodbyeDPI-UI/releases/tag/1.2.3)

**Reply by LordArrin** [reply]
Правда при штатном обновлении выдало ошибку при распаковке какого то dll, пришлось устанавливать поверх, тогда всё заработало. Но настройки сбросились, пришлось настраивать заново.
